packages/maskbook/src/plugins/Trader/trader/0x/useTradeComputed.ts

```diff
@@ -34,7 +33,7 @@ export function useTradeComputed(
             // minimumProtocolFee
             priceImpact: new BigNumber(0),
 
-            trade_: trade,
+            trade_: { ...trade, buyAmount: outputAmount.toFixed() },
```

Comment:
Wow, this is really important! 

<img src="https://user-images.githubusercontent.com/63733714/108686841-ee30fd00-7530-11eb-801a-e2982e3ca235.png" width=500 />

<img src="https://user-images.githubusercontent.com/63733714/108686896-0143cd00-7531-11eb-966b-7bcf9b928ba0.png" width=500 />

<img src="https://user-images.githubusercontent.com/63733714/108686959-13257000-7531-11eb-825e-e9db96f849d6.png" />